### ISSUE TITLE
Error when provisioning multiple chains

### ISSUE THREAD

**Issue by phahulin** [poanetwork/blockscout-terraform#31]
When provisioning infra with multiple chains, I got the following error:
```
data.aws_iam_policy_document.instance-assume-role-policy: Refreshing state...
data.aws_iam_policy_document.deployer-assume-role-policy: Refreshing state...
data.aws_availability_zones.available: Refreshing state...
data.aws_ami.explorer: Refreshing state...
data.aws_iam_policy.AmazonEC2RoleForAWSCodeDeploy: Refreshing state...
data.aws_iam_policy.AmazonEC2RoleForSSM: Refreshing state...
data.aws_iam_policy_document.config-policy: Refreshing state...
data.aws_iam_policy.AWSCodeDeployRole: Refreshing state...

------------------------------------------------------------------------
Releasing state lock. This may take a few moments...

Error: Error running plan: 2 error(s) occurred:

* module.stack.aws_autoscaling_policy.explorer-up: 1 error(s) occurred:

* module.stack.aws_autoscaling_policy.explorer-up: Resource 'aws_autoscaling_group.explorer' not found for variable 'aws_autoscaling_group.explorer.name'
* module.stack.aws_autoscaling_policy.explorer-down: 1 error(s) occurred:

* module.stack.aws_autoscaling_policy.explorer-down: Resource 'aws_autoscaling_group.explorer' not found for variable 'aws_autoscaling_group.explorer.name'

```

To include multiple chains I added the following lines to `terraform.tfvars`:
```
chains = {
    "sokol" = "https://sokol.poa.network"
    "core" = "https://core.poa.network"
}

chain_trace_endpoints = {
    "sokol" = "https://sokol-trace.poa.network"
    "core" = "https://core-trace.poa.network"
}
```

**Comment by acravenho** [comment]
I'm also receiving a similar error here when deploying multiple chains.

**Comment by acravenho** [comment]
Fixed by #49
